pkg/commands/alerts/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package alerts contains commands to inspect the CLI configuration.
```

Comment:
This description sounds wrong.

I imagine it should be more like:

```go
// Package alert contains commands to inspect and manipulate Fastly Alerts.
```

Reply:
Done